src/components/DateInput.vue

```diff
@@ -156,17 +147,25 @@ export default {
       this.$emit('close')
       this.isFocusedUsed = false
     },
+    /**
+     * Toggles the calendar (unless `show-calendar-on-button-click` is true)
+     */
     handleInputClick() {
+      /* istanbul ignore next */
```

Comment:
I fully agree that it's not necessary to have 100% coverage. However............... how are you to know if there is a gap in your tests for a new feature you have just implemented? Are you supposed to remember which lines were not covered previously? Adding these ugly istanbul ignore lines helps you to see - at a glance - whether your new commit needs any additional tests.

Reply:
You know that there is a gap for a new feature if you add the new feature. 
For example if you add a function to add a suffix to a string you need to write a test that this suffix is correctly appended without looking at the coverage.
Basically every line you add/touch needs a corresponding unit test, which is why Test driven Development is praised a lot.

It's just in the real world you don't need to test every single line because some lines are behind super edge cases where you would need to write so much code just to test this little line

Reply:
I'm yet to fully espouse TDD, but am definitely on the way there... In the meantime, I've got an 'end goal' that I'm moving towards and am trying to 'write up' - as nicely as possible - the various steps (i.e. commits) along the path to get there.

Therefore, I find it helpful to know - for each commit - whether the tests I've written are sufficient. Having a 'clean slate' with 100% test coverage makes it easier for me to do that.

Let's turn this on its head: what's the benefit of not adding these istanbul ignore lines? You say it will 'distort the test coverage'. But who cares whether the total test coverage is 97.6% or 96.8%. Isn't this just a tool to help you ensure that your code is adequately tested? Your last line suggests that you agree that there's no need to test every single line, so why not just ignore those lines?

Reply:
The test coverage is not something only for the developer of an open source library. It helps other developer do decide if they should use the library or help out. If you hide the fact that you don't test everything and someone sees it, the trust in the project will fall rapidly because outsiders won't know how much of your code is ignored in the tests.

Furthermore if the code base grows and you are adding more and more of those ignore comments the real coverage will drop without you ever notice how much it really drops, which is pretty risky because over 90% is a good enough coverage but below is pretty bad.
If you need 100% you can see the current value as 100%. Currently in the master branch the code coverage is 99.84, if it drops below it your changes needs more tests.

Reply:
OK, I accept this trust argument :-)

What's the best way to proceed? Do I drop the offending commit and force push, or should I add another commit to remove the istanbul lines... or doesn't it matter?

Reply:
You can just add a new commit, `test`, `chore`, `refactor` commits won't be added to the changelog.

Reply:
OK thanks. I've done that and also added another minor refactor as part of this PR. Cheers!